chapters/03-funcoes.md

```diff
@@ -75,7 +75,7 @@ console.log(x);
 // → inside f2
 ```
 
-Esse comportamento ajuda a prevenir interferências acidentais entre funções. Se todas as variáveis fossem compartilhadas por toda a aplicação, seria muito trabalhoso garantir que o mesmo nome não fosse utilizado em duas situações com propósitos diferentes. Além disso, *se* fosse o caso de reutilizar uma variável com o mesmo nome, talvez você pudesse se deparar com efeitos estranhos de códigos que alteram o valor da sua variável. Assumindo que variáveis locais existem apenas dentro do contexto da função, a linguagem torna possível ler e entender funções como “pequenos universos”, sem termos que nos preocupar com o código da aplicação inteira de uma só vez.
+Esse comportamento ajuda a prevenir interferências acidentais entre funções. Se todas as variáveis fossem compartilhadas por toda a aplicação, seria muito trabalhoso garantir que o mesmo nome não fosse utilizado em duas situações com propósitos diferentes. Além disso, se fosse o caso de reutilizar uma variável com o mesmo nome, talvez você pudesse se deparar com efeitos estranhos de códigos que alteram o valor da sua variável. Assumindo que variáveis locais existem apenas dentro do contexto da função, a linguagem torna possível ler e entender funções como pequenos universos, sem termos que nos preocupar com o código da aplicação inteira de uma só vez.
```

Comment:
:-1: na remoção do itálico no "se"
 
O trecho original: ![](https://s3.amazonaws.com/f.cl.ly/items/2l060q2l2A3g2M0i0726/Screen%20Shot%202015-05-01%20at%203.21.23%20AM.png)

:+1: no trecho `[...] pequenos universos, [...]`